browser/mapping.go

```diff
@@ -211,7 +215,10 @@ func mapElementHandle(vu k6modules.VU, eh api.ElementHandle) mapping {
 //
 //nolint:funlen
 func mapFrame(vu k6modules.VU, f api.Frame) mapping {
-	rt := vu.Runtime()
+	var (
+		rt  = vu.Runtime()
+		ctx = vu.Context()
```

Comment:
Do we need to create a new context in each mapping method?

Reply:
This doesn't create a new context but gets the existing wrapped context from`vu.Context()`. So it's the same context we set [here](https://github.com/grafana/xk6-browser/blob/3a4bee6af98006b544ae0a5af1267d23b1c472da/browser/module.go#L46-L50) only once. Were you asking this, or maybe I didn't get what you meant? 🤔

Reply:
@ankur22 please check [this](https://github.com/grafana/xk6-browser/pull/704/commits/089689d5e8a8b4422418b7abec109b55bd5399f4) out and tell me WDYT.

Reply:
@inancgumus yes, that's what i think we should be doing. WDYT?

Reply:
Ah, I got you now. You're right. I'll fix it.

Reply:
Check [this](https://github.com/grafana/xk6-browser/pull/704/commits/5822656b1d4f06f92eae38eb8334122b54d39afe) out.

Reply:
Yeah, i think that makes more sense, and is easier to reason with. 👍